Test Monitor/OEE/OEE.fpg

```diff
@@ -114,733 +84,80 @@
+{"name": "tile6", "type": "graph", "x": "1160", "y": "989.0625", "width": "1154", "height": "655.375", "attributes": [{"prefix": "tile"}, {"title": {"localizationKey":"tile6.title"}}, {"multiDatasource": "%5B%7B%22label%22%3A%220%22%2C%22value%22%3A%22%7B%5C%22type%5C%22%3A%5C%22notebook%5C%22%2C%5C%22binding%5C%22%3A%5C%22%7B%5C%5C%5C%22name%5C%5C%5C%22%3A%5C%5C%5C%22_shared/reports/OEE%20Health.ipynb%5C%5C%5C%22%2C%5C%5C%5C%22executionInterval%5C%5C%5C%22%3A86400%2C%5C%5C%5C%22version%5C%5C%5C%22%3A2%2C%5C%5C%5C%22output%5C%5C%5C%22%3A%5C%5C%5C%22health_graph%5C%5C%5C%22%2C%5C%5C%5C%22params%5C%5C%5C%22%3A%5B%7B%5C%5C%5C%22name%5C%5C%5C%22%3A%5C%5C%5C%22group_by%5C%5C%5C%22%2C%5C%5C%5C%22value%5C%5C%5C%22%3A%5C%5C%5C%22Day%5C%5C%5C%22%7D%2C%7B%5C%5C%5C%22name%5C%5C%5C%22%3A%5C%5C%5C%22results_filter%5C%5C%5C%22%2C%5C%5C%5C%22value%5C%5C%5C%22%3A%5C%5C%5C%22startedWithin%20%3C%3D%20%5C%5C%5C%5C%5C%5C%5C%2230.0%3A0%3A0%5C%5C%5C%5C%5C%5C%5C%22%5C%5C%5C%22%7D%5D%7D%5C%22%7D%22%7D%5D"}, {"renderStyle": "line"}, {"xAxisLabel": {"localizationKey":"tile6.xAxisLabel"}}, {"yAxisLabel": {"localizationKey":"tile6.yAxisLabel"}}, {"yAxisAutoScale": "grow"}, {"yAxisMin": "0"}, {"yAxisMax": "100"}, {"windowSize": "3600"}, {"navigationURLEnabled": "false"}, {"plotLegend": "%5B%7B%22label%22%3A%7B%22localizationKey%22%3A%22tile6.plotLegend.label0%22%7D%2C%22color%22%3A%22rgba%280%2C%20159%2C%20245%2C%200.9%29%22%7D%5D"}, {"zIndex": "6"}, {"columnRow": "%5B6%2C6%5D"}, {"gridSpan": "%5B6%2C4%5D"}, {"plotLegendEnabled": "false"}, {"xAxisLabelEnabled": "true"}, {"yAxisLabelEnabled": "true"}, {"plotLegendEditable": "false"}, {"plotLegendVisible": "false"}],
+"bindings": [{"value": [{"type":"notebook","binding":"{\"name\":\"_shared/reports/OEE Health.ipynb\",\"executionInterval\":86400,\"version\":2,\"output\":\"health_graph\",\"params\":[{\"name\":\"group_by\",\"value\":\"Day\"},{\"name\":\"results_filter\",\"value\":\"startedWithin <= \\\"30.0:0:0\\\"\"}]}"}]}]},
+{"name": "tile3", "type": "progress", "x": "2", "y": "0", "width": "575", "height": "325.6875", "attributes": [{"prefix": "tile"}, {"title": {"localizationKey":"tile3.title"}}, {"datasource": "%7B%22type%22%3A%22notebook%22%2C%22binding%22%3A%22%7B%5C%22name%5C%22%3A%5C%22_shared/reports/OEE%20Health.ipynb%5C%22%2C%5C%22executionInterval%5C%22%3A86400%2C%5C%22version%5C%22%3A2%2C%5C%22output%5C%22%3A%5C%22availability_scalar%5C%22%2C%5C%22params%5C%22%3A%5B%7B%5C%22name%5C%22%3A%5C%22group_by%5C%22%2C%5C%22value%5C%22%3A%5C%22Day%5C%22%7D%2C%7B%5C%22name%5C%22%3A%5C%22results_filter%5C%22%2C%5C%22value%5C%22%3A%5C%22startedWithin%20%3C%3D%20%5C%5C%5C%2230.0%3A0%3A0%5C%5C%5C%22%5C%22%7D%5D%7D%22%7D"}, {"progressDisplayFormat": "horizontal"}, {"denominator": "100"}, {"navigationURLEnabled": "false"}, {"useFilter": "false"}, {"zIndex": "12"}, {"columnRow": "%5B0%2C0%5D"}, {"gridSpan": "%5B3%2C2%5D"}],
```

Comment:
Did you update your notebook to change the value returned? If not, you probably need to change the denominator for all your progress tiles to 1 instead of 100.